catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardImpl.java

```diff
@@ -147,7 +154,7 @@ public MetacardImpl(Metacard metacard) {
      * given {@link Metacard}.
      *
      * @param metacard the {@link Metacard} to create this new {@code Metacard} from
-     * @param type the {@link MetacardType} of metacard to create
+     * @param type     the {@link MetacardType} of metacard to create
```

Comment:
I think the formatter was the cause of this, not sure why it wasn't caught when the whole code base was formatted